open_prices/prices/models.py

```diff
@@ -183,15 +183,15 @@ class Price(models.Model):
 
     price = models.DecimalField(
         max_digits=10,
-        decimal_places=2,
+        decimal_places=3,
```

Comment:
@raphael0202 should we restrict to 2 for most of the currencies ?

Reply:
todo in a following PR (not priority)

Reply:
so fyi there was an issue in the migration in prod
one price had `price_without_discount=10000100.00` : 10 digits, 2 decimal places. so already the max. so impossible to migrate as this PR adds a decimal place but without adding a new digit...